### PR TITLE
Fashion Update: Earrings & Makeup

### PR DESCRIPTION
**Pull request by angelofallars**
# Description

Adds **38** new earrings, **31** new face markings, and **16** bracelet/watch markings, based from Aurorastation cosmetics. Each earring, bracelet, and watch marking is applied to just one side of the body, allowing for greater customizability and mixing-and-matching markings.

Most notably, the new face markings unlock **full customization of every pixel** in the 5x4 pixel space of the face. Examples of this are shown below.

Arachne now have access to **more than a hundred**  markings above the leg, after not having any markings in their 2 years of existence.

### Pauline

**Head (Side):** Drop Earrings (Left), Drop Earrings (Right)
**Left Hand:** Left Bracelet

### Red and Blue

**Head:** Cheek Spot (Left), Eye Corner (Left), Cheek Spot (Right), Eye Corner (Right)
**Head (Side):** Double Gemstone Earrings (Left), Double Gemstone Earrings (Right)
**Left Hand:** Left Watch (Colorable)
**Left Arm:** Left Arm Bracelet
**Right Hand:** Right Watch (Colorable)
**Right Arm:** Right Arm Bracelet

### Pink

**Head (Side):** Hoop Earrings (Left), Double Gemstone Earrings (Right)
**Left Hand:** Left Bracelet
**Left Arm:** Left Arm Bracelet
**Right Hand:** Right Bracelet
**Right Arm:** Right Arm Bracelet

### Psychic

**Head:** Bindi, Full Blush
**Head (Side):** Long Crystal Earrings (Left), Long Crystal Earrings (Right)
**Left Hand:** Left Bracelet
**Right Hand:** Right Bracelet
**Right Arm:** Right Arm Bracelet

### Cyclops

**Head [Eyes]:** Bindi, Glabella, Nose Tape
**Head [Cover for mouth area]:** Lip Corner (Left), Lip Corner (Right), Cheek Spot (Left), Cheek Spot (Right), Lips
**Head (Side):** Cross Earrings (Left), Cross Earrings (Right)

### Arachne

image

**Head:** Left Arachne Eye, Right Arachne Eye, Lips, Neck Cover (Slim Thick)
**Head (Side):** Long Crystal Earrings (Left), Long Crystal Earrings (Right)
**Left Arm:** Left Robotic Arm from Hesphiastos Industries
**Left Hand:** Left Robotic Arm from Bishop Cybernetics
**Right Arm:** Right Arm Bracelet
**Right Hand:** Right Watch (Colorable), Right Nail Polish

## Technical Details

I did a quick fix in [MarkingsSet.cs](https://github.com/Simple-Station/Einstein-Engines/compare/master...angelofallars:Einstein-Engines:fashion-update-markings?expand=1#diff-54373833c74b35579f9b3293d5bb0c17eda0d214c11f27dcb40c334ef8c6b513) to handle a list index out of bounds error where the number of default markings was lower than the maximum allowed markings, like with Onis who have a default horn but can wear 2 earrings.

# Changelog

:cl: Skubman
- add: 80+ new markings have arrived, including earrings, makeup, bracelets, and watches. Sashay over to Character Setup to personalize your character like never before!
- add: The Arachne species can now select markings above the leg, including cybernetics, makeup, tattoos, noses, earrings, heterochromia, bracelets, gauze, and more!
- tweak: The available points for Head (Side), Left Hand, and Right Hand markings have been increased for most species to support the new markings.
- tweak: The Nail Polish markings have been moved from the Overlay category to the Left Hand and Right Hand categories.

### CODE REVIEW

**Comment by DEATHB4DEFEAT** [comment]
I love the looks of these characters you keep making

**Comment by VMSolidus** [comment]
image

THAT WAS A BUG THIS WHOLE TIME? GUH I CAN'T BELIEVE I NEVER BOTHERED LOOKING AT THE MARKINGS CODE ABOUT IT. GUH.

**Comment by VMSolidus** [comment]
>Go to click comment
>Accidentally click close with comment
Truly my fat fingers knows no bounds.

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/earrings.yml [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+          color: "#f3d052"
+  sprites:
+  - sprite: Mobs/Customization/earrings.rsi
+    state: bangle_l
+
+- type: marking
+  id: EarringsBangleRight
+  bodyPart: HeadSide
+  markingCategory: HeadSide
+  speciesRestriction: [Dwarf, Human, SlimePerson, Arachnid, Reptilian, Diona, Oni, Felinid, Vulpkanin, Harpy, Gingerbread]
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/earrings.yml:195]: I hate that we can't just do the inverse of this by making a species Blacklist, for when you have a marking that could be put on any species. 

Also, these are missing Arachne!

**Review by VMSolidus — CHANGES_REQUESTED** [review]

**Comment by angelofallars** [comment]
@VMSolidus Arachne markings are here.
image

**Review by VMSolidus — APPROVED** [review]

**Review by sleepyyapril — APPROVED** [review]

**Pull request merged**